### ISSUE TITLE
question exonuclease/phosphodiesterase

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11520]
How do we correctly describe:

Uniprot
3'-5' exoribonuclease that posttranscriptionally removes uridine and adenosine nucleosides from the 3' end of spliceosomal U6 small nuclear RNA (snRNA), directly catalyzing terminal 2', 3' cyclic phosphate formation

SGD
Putative phosphodiesterase specific for U6 snRNA 3' end modification; trims the 3' poly(u) tract to leave a terminal 3' phosphate; human homolog, hUSB1 (aka C16orf57) produces a 2',3' cyclic phosphate; 

I see it variously described as:

GO:0000175 3'-5'-exoribonuclease activity http://www.uniprot.org/uniprot/Q9BQ65 USB1 human

GO:0008081 phosphoric diester hydrolase activity  SGD USB1

Should there be a more specific child term to capture both of these terms?

Should there be a grouping term here for nucleotide phosphodiesterases?
http://www.ebi.ac.uk/QuickGO/GSearch?q=phosphodiesterase

Reported by: ValWood

Original Ticket: [geneontology/ontology-requests/11346](https://sourceforge.net/p/geneontology/ontology-requests/11346)

**Comment by ValWood** [comment]
So it seems that this one can be closed now?

**Comment by ValWood** [comment]
My query in this one  was fixed independently by Rob Nash yesterday.
==

Hi Val,

I took a look at your annotation and the paper. It seems that the older paper we had used came to a different conclusion based on the evidence that had at the time. However, I think the evidence presented in the paper you used was solid and I think your choice of term was good, so I have removed our current function annotation, and replaced if with an ISS to mpn1. I have also added the paper to our db and updated our description. I don’t think the two terms should be related, so I don’t think one should be an ancestor of the other. 

Cheers,
Rob
